File: beecrowd/1/1008/1008.go
package main

import (
	"fmt"
)

func main() {
	var numero, horas int
	var valor float64

	fmt.Scanf("%d\n%d\n%f", &numero, &horas, &valor)
	fmt.Printf("NUMBER = %d\n", numero)
	fmt.Printf("SALARY = U$ %.2f\n", float64(horas)*valor)

}

File: beecrowd/1/1008/1008.cs
using System;

class URI {

    static void Main(string[] args) {
        int numero, horas;
        double valor;
        numero = int.Parse(Console.ReadLine());
        horas = int.Parse(Console.ReadLine());
        valor = double.Parse(Console.ReadLine());
        Console.WriteLine("NUMBER = {0}", numero);
        Console.WriteLine("SALARY = U$ {0}", (horas * valor).ToString("0.00"));
    }
}
